.github/workflows/main.yml

```diff
@@ -17,16 +17,18 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python: 3.9
-            image_tag: nightly
-          - python: 3.6
-            image_tag: latest
-          - python: 3.9
-            image_tag: 3.9
-          - python: 3.7
-            image_tag: 3.8
-          - python: 3.8
-            image_tag: 3.7
+          - python: "3.9"
+            image_tag: "nightly"
+          - python: "3.6"
+            image_tag: "3.11"
+          - python: "3.6"
+            image_tag: "3.10"
```

Comment:
I hate YAML!
Not in general, but this automated type guessing is driving me mad.